tests/bor/testdata/genesis.json

```diff
@@ -8,13 +8,25 @@
     "eip158Block": 0,
     "byzantiumBlock": 0,
     "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "muirGlacierBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
```

Comment:
If this change involves changing the genesis this much I would wait until we have the new CLI in use. Otherwise, we have to coordinate a change over the launch repo as well. In the new CLI the genesis file is embed.

Reply:
This is a test file where all the changes were present. So I have included all the changes and fields till now.
Fields which will be included - londonBlock, period structure change, backupMultiplier structure change, burntContract

Reply:
Let me know should we wait or go ahead with it